docs/src/css/components/_breadcrumb.scss

```
@@ -0,0 +1,189 @@
+nav[aria-label='Breadcrumb'] {
+  display: flex;
+  align-items: center;
+
+  width: fit-content;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)